coprocess/python/dispatcher.py

```diff
@@ -42,19 +43,11 @@ def find_middleware(self, path):
         return found_middleware
 
     def load_bundle(self, base_bundle_path):
-        bundle_path = path.join(base_bundle_path, '*.py')
-        bundle_modules = self.get_modules(bundle_path)
-        sys.path.append(base_bundle_path)
-        for module_name in bundle_modules:
-            middleware = self.find_middleware(module_name)
-            if middleware:
-                middleware.reload()
-            else:
-                middleware = TykMiddleware(module_name)
-                self.middlewares.append(middleware)
+        bundle_path = path.join(base_bundle_path, 'middleware.py')
```

Comment:
@lonelycode This line will enforce `middleware.py`, it's useful to avoid collisions with other package/file names and it's also required to use the alternative `importlib` loader function that I've added in `middleware.py`, if the user loads a bundle that doesn't have `middleware.py` a Python error will show up (won't panic). All our samples use `middleware.py`.

Reply:
In case you think this is too strict for a patch, I could try to modify the code again to "scan" all the Python files and load the appropriate one -the one containing the hooks-, just like the previous functionality.

Reply:
Sounds ok to me